mmcv/runner/dist_utils.py

```diff
@@ -12,6 +12,9 @@
 from torch._utils import (_flatten_dense_tensors, _take_tensors,
                           _unflatten_dense_tensors)
 
+if hasattr(torch, 'is_mlu_available') and torch.is_mlu_available():
```

Comment:
line 15 can be removed and move line 16 to line 54.

Reply:
Done. Use IS_MLU instead and move import into functions.